### DISCUSSION TITLE
Please tell me your favorite programming language.

### DISCUSSION THREAD

**Discussion by tomoyasuzuki** [NAIST-SE/AttractingContributors#924] [2023 Attraction Contributions]
Hi everyone, I started my project that is intended to decide the best popular programming language and understand why the language is so popular. please visit https://github.com/tomoyasuzuki/AttractingContributors-Programming-Language and try to add your favorite programming language !!

**Comment by rfLENtlr** [comment]
I like Python.
Send a pull request.
Would you like to contribute to my repository as well?
https://github.com/rfLENtlr/AttractingContributors-OS

**Comment by bconfessor** [comment]
I like C#, I sent you a pull request! 
Could you please tell me your favorite song?? 
https://github.com/bconfessor/FavoriteSong/blob/main/README.md

**Comment by ppjaisri** [comment]
I love Python
I sent you a pull requests
Would you like to make Minecraft totally epic ?
https://github.com/ppjaisri/AttractingContributors-Install-add-on-to-Minecraft
